python/hail/__init__.py

```diff
@@ -2,7 +2,7 @@
 from hail.representation import *
 from hail.context import HailContext
 from hail.dataset import VariantDataset
-from hail.expr import *
+from hail.htypes import *
```

Comment:
I don't like putting h[ail] as a prefix on things already in the hail package.  I prefer `typ` or `type_` or `types` or something.  The Python style guide:

https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles

says, "single_trailing_underscore_: used by convention to avoid conflicts with Python keyword, e.g. `Tkinter.Toplevel(master, class_='ClassName')`".